.changeset/fair-cats-tickle.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-css': patch
+---
+
+Remove unused CSS class.
```

Comment:
I think we can just omit this changeset, it's too small to list out.

Reply:
Sounds good, deleted.